requirements.txt

```diff
@@ -9,13 +9,14 @@ enum34; python_version == '2.7'
 fauxfactory>=2.0.7
 google-api-python-client
 inflection
-miq-version==0.1.4
+miq-version>=0.1.5
```

Comment:
shouldn't merge this, I just haven't released miq_version yet and locally this helps, will freeze on a version after releasing there first.